trollflow2/plugins/__init__.py

```diff
@@ -119,12 +119,15 @@ def resample(job):
             maxarea = get_config_value(product_list,
                                        '/product_list/areas/' + str(area),
                                        'use_max_area')
+            native = conf.get('resampler') == 'native'
```

Comment:
could we make this generic to support other resamplers like ewa or gradient ? and pass their kwargs too ?

Reply:
The basic calls to other resampler are already handled in the non-`None` branch (`else:`) of the resampler. Although, the non-kdtree based resample kwargs need to be handled somehow, currently there's only handling for `radius_of_influence`, `cache_dir` etc. I think that is best done when we get the gradient resampler merged in pyresample and satpy.

Reply:
ok